### DISCUSSION TITLE
Assigning write permissions with tags not working

### DISCUSSION THREAD

**Discussion by scott-oit** [requarks/wiki#3320] [Error / Bug Report]
**Describe the bug**
Assigning write permissions for a group based on tags gives the user errors when trying to save. The user has the pencil icon to edit, but when they type or save they get a warning of "An unexpected error occurred". The group has full global content permissions.

**To Reproduce**
..**Create test group**
1. Create test group `test-group`
2. Assign 'test-group' full content rights in PERMISSIONS
3. Clear out PAGE RULES
4. Create new PAGE RULE with full permissions - Tag Matches... - `test-tag`
5. UPDATE GROUP

..**Change User Permissions**
1. Edit test user
2. Remove any groups
3. Add `test-group`

..**Tag document**
1. Tag test document with `test-tag`

..**Test**
Sign in as user and attempt to edit and save page. You should be able to open the editor, but not save.

**Expected behavior**
Assigning the write permissions to a tag should let the user edit the page.

**Screenshots**
NA

**Host Info (please complete the following information):**
 - OS: Docker
 - Wiki.js version: 2.5.170
 - Database engine: postgress 11.6

**Additional context**

**Comment by rrg92** [comment]
I confirmed that bug. The problem is with the "IsConflic" check that page was.
It check for "manage:pages" and "read:pages" permissions, but don't pass the page tags. Just pass the page path and locale.

![image](https://user-images.githubusercontent.com/16887793/190722765-da6b201b-7aee-44c7-b51c-211bf8883d04.png)


Workaround is use a rule that just use path, like regular expression.

**Reply by Coros** [reply]
+1 here.  Trying to find a replacement for Confluence and thought this might be the one.  I don't see how maintaining path rules would work for us.  We have hundreds of staff and a good amount of groups that need to have a manageable access control system.
